packages/edit-site/src/components/sidebar-navigation-item/style.scss

```diff
@@ -1,4 +1,5 @@
-.edit-site-sidebar-navigation-item.components-item {
+.edit-site-sidebar-navigation-item.components-item,
+.edit-site-sidebar-navigation-screen__content .edit-site-navigation-inspector .components-button {
```

Comment:
Why do we need a specific style here in a generic component?

Reply:
I removed the styles for .edit-site-sidebar-navigation-item.components-item, the other styles are required so the buttons looked "themed" on the dark mode.